jsk_recognition_utils/node_scripts/polygon_array_publisher.py

```diff
@@ -1,21 +1,60 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import numpy as np
 import rospy
-from geometry_msgs.msg import PolygonStamped, Point32
+from geometry_msgs.msg import Point32, PolygonStamped
+from jsk_recognition_msgs.msg import ModelCoefficientsArray
 from jsk_recognition_msgs.msg import PolygonArray
+from pcl_msgs.msg import ModelCoefficients
+
+
+def is_planar(points):
```

Comment:
It would be helpful if you could write a summary or comment about the algorithm.

For example, I can easily understand this web page.
http://www.not-enough.org/abe/manual/argo/pointOnPlane.html

Reply:
I see. Why don't you write a comment for understanding?

Reply:
OK, I will try.

Reply:
I created PR
https://github.com/iory/jsk_recognition/pull/13